.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: renovate
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: dry-run
+        required: false
+        type: string
+        default: "false"
+      log-level:
+        description: log-level
+        required: false
+        type: string
+        default: info
+  schedule:
+    - cron: "0 */3 * * 1-4"
+  push:
+    branches:
+      - main
+    paths:
+      - .github/renovate.json5
+      - .github/renovate/**.json5
+      - .github/workflows/renovate.yaml
+
+concurrency:
+  group: renovate
+  cancel-in-progress: true
+
+jobs:
+  renovate:
+    uses: strg-at/github-workflows/.github/workflows/run-renovate.yaml@c3b65900e045c388e6895d74522317427453fde0 # v1.6.1
```

Comment:
Makes sense to already set the 1.7 tag no?

Reply:
want to test automerge